src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
Is this file added to the template on purpose? Is it related to SAF debug mode?

Reply:
@asbfall yes.